frontend/src/ramp.pug

```diff
@@ -17,6 +17,26 @@
           }"
       )
 
+  template(v-else-if="zoomfromramp")
```

Comment:
Instead of adding another `v-else-if` template that consists of mostly duplicate code, how about just add on to the `v-else` template where it looks smth like this:

`v-bind:href="zoomfromramp ? getLink(user, slice)", target="_blank" : javascript:void(0)`
`v-on:click="zoomfromramp ? {} : openTabZoom(user, slice)"`